Project.toml

```diff
@@ -21,10 +21,11 @@ Requires = "0.5.2, 1"
 julia = "1"
 
 [extras]
+Aqua = "4c88cf16-eb10-579e-8560-4a9242c79595"
 IntervalArithmetic = "d1acc4aa-44c8-5952-acd4-ba5d80a2a253"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [targets]
-test = ["IntervalArithmetic", "LinearAlgebra", "SparseArrays", "Test"]
+test = ["IntervalArithmetic", "LinearAlgebra", "SparseArrays", "Test", "aqua"]
```

Comment:
I think `"aqua"` here refers to the module, so it should be `"Aqua"`. See also [this line](https://github.com/JuliaDiff/TaylorSeries.jl/actions/runs/5236879611/jobs/9454701672?pr=329#step:5:24) from the tests.

Reply:
Yes, thank you!